packages/mds-repository/repository.ts

```diff
@@ -228,6 +240,13 @@ export const ReadWriteRepository = {
 
     const revertAllMigrations = async (): Promise<void> => revertAllMigrationsUsingConnection(await connect('rw'))
 
+    const truncateAllTables = async (options: Partial<{ unsafe: true }> = {}): Promise<void> => {
+      if (!options.unsafe) {
```

Comment:
Do we have a use case in mind for `unsafe`?

Reply:
`REPL`. I wanted to make it available, but require it to be intentional. I wouldn’t expect this to be called from  a service.

Reply:
Makes sense and I could see it being handy. I'm just thinking about security of course but I guess if someone's already in the REPL that ship has sailed.